store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -137,6 +141,31 @@ public void initialise(final Schema schema, final StoreProperties properties)
         TableUtils.ensureTableExists(this);
     }
 
+    @Override
+    public void validateSchemas() {
+        super.validateSchemas();
+
+        final ValidationResult validationResult = new ValidationResult();
+
+        getSchemaElements().entrySet().forEach(entrySet -> entrySet.getValue().getProperties().forEach(propertyName -> {
+            final Serialiser serialisation = entrySet.getValue().getPropertyTypeDef(propertyName).getSerialiser();
+
+            if (serialisation != null && !(serialisation instanceof ToBytesSerialiser)) {
```

Comment:
As discussed: This will be moved into Store removing duplication, and will appear before the Exception is thrown for null schema and will need to be null safe.